File: ClientApp/ClientApp/Models/Responses/JoinRoomResponse.cs
using Newtonsoft.Json;
using ClientApp.Models.ResponseErrors;

namespace ClientApp.Models.Responses
{
    class JoinRoomResponse : Response
    {
        private JoinRoomResponse() { }
        public override ResponsesCodes GetCode() => ResponsesCodes.JoinRoomResponse;
        [JsonProperty]
        public JoinRoomResponseErrors Errors { get; private set; }

    }
}

File: M_Trivia/M_Trivia/JoinRoomResponse.cpp
#include "JoinRoomResponse.hpp"
#include "json.hpp"


JoinRoomResponse::JoinRoomResponse(JoinRoomResponseErrors* errors) 
	: Response(errors)
{
}
	
ResponseCodes JoinRoomResponse::getCode() const 
{
	return ResponseCodes::C_JoinRoomResponse; 
}




